test/consul/gen-cert.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+
+need_regen() {
+	files="server.crt server.key server-ca.crt consul-client.crt consul-client.key consul-client-ca.crt cilium-consul.yaml"
+	for file in $files; do
+		if [ ! -f $file ]; then
+			return -1
+		fi
+	done
+	return 0
+}
+
+clean() {
+	rm  -f server.crt server.key server-ca.crt consul-client.crt consul-client.key consul-client-ca.crt cilium-consul.yaml
+	exit 0
+}
+
+gen_consul_config() {
+	cat > cilium-consul.yaml <<EOF
+---
+cafile: '$dir/consul-client-ca.crt'
+keyfile: '$dir/consul-client.key'
+certfile: '$dir/consul-client.crt'
+EOF
+}
+
+gen() {
```

Comment:
Your intention is to generate certificates every time when tests are running? If yes, please remove all *.crt, *.key and *.pem files from this commit and add them to `.gitignore`.

Also, I don't see this script being called anywhere. Should developers use it manually?

Reply:
neh, the script is there if needed later on. certs are part of the PR